src/duration-picker/ForwardStartingOptions.js

```diff
@@ -43,6 +43,9 @@ export default class ForwardStartingOptions extends PureComponent {
     onTimeChange = e => {
         const { dateStart } = this.props;
         const inputValue = returnValidTime(e.target.value, ':');
+        if (isValidTime(inputValue)) {
+             showError('Time format is wrong!');
```

Comment:
@borisyankov i actually wanted to avoid that even though it would better though in this case. T o use the component, we might have to use the component state object.